composer.json

```diff
@@ -63,15 +49,37 @@
             "ZendDeveloperToolsTest\\": "test/"
         }
     },
+    "config": {
+        "sort-packages": true
+    },
+    "extra": {
+        "branch-alias": {
+            "dev-master": "1.1.x-dev",
+            "dev-develop": "1.2.x-dev"
+        },
+        "zf": {
+            "module": "ZendDeveloperTools"
+        }
+    },
     "scripts": {
         "check": [
-            "@cs",
+            "@cs-check",
             "@test"
         ],
-        "coveralls": "coveralls",
-        "cs": "phpcs",
+        "cs-check": "phpcs",
         "cs-fix": "phpcbf",
-        "test": "phpunit",
-        "test-coverage": "phpunit --coverage-clover clover.xml"
-    }
+        "test": "phpunit --colors=always",
+        "test-coverage": "phpunit --colors=always --coverage-clover clover.xml"
+    },
+    "authors": [
```

Comment:
We can remove authors, as we did in other repos, git log should be enough.

Reply:
I left these here as these authors started the project before it came to the zendframework org.